ini/stm32f4.ini

```diff
@@ -423,12 +419,10 @@ build_flags       = ${stm_flash_drive.build_flags} ${stm32f4_I2C1.build_flags}
 
 #
 # MKS Robin Nano V3 with USB Flash Drive Support and Shared Media
-# Currently, using a STM32duino fork, until USB Host and USB Device MSC get merged
 #
 [env:mks_robin_nano_v3_usb_flash_drive_msc]
 platform          = ${common_stm32.platform}
 extends           = env:mks_robin_nano_v3
-platform_packages = framework-arduinoststm32@https://github.com/rhapsodyv/Arduino_Core_STM32/archive/usb-host-msc-cdc-msc-3.zip
```

Comment:
that's a regression as https://github.com/rhapsodyv/Arduino_Core_STM32/commit/21f67b2c3928eb9e046b79e8ff701a21dc337383 still wasn't merged in arduino_core_STM32 and we will still need a fork and a rebase on top of 2.0.0

Reply:
Ah, I incorrectly assumed 14.0 had those features.